docs/api-aggregation-setup.md

```diff
@@ -211,5 +212,12 @@ Before installing the helm chart, run the script in contrib by
 `source`ing it, to define all of the necessary variables.
 
 ```shell
-source /contrib/svc-cat-apiserver-aggregation-tls-setup.sh
+source charts/catalog/aggregation/tls-setup/sh
+```
+
+To delete, all resources, execute the following:
+
+```console
+helm delete --purge catalog
+k delete apiservice v1alpha1.servicecatalog.k8s.io
```

Comment:
not sure we can expect everyone to have kubectl aliased to k. Can we explicitly right out kubectl?